lib/boba/enums_railtie.rb

```diff
@@ -0,0 +1,100 @@
+# typed: true
+# frozen_string_literal: true
+
+require "rails/railtie"
+
+module Boba
+  class EnumsRailtie < Rails::Railtie
+    railtie_name(:boba)
+
+    initializer("boba.add_enum_classes") do
+      ActiveSupport.on_load(:active_record) do
+        next if defined?(ActiveRecordTypedEnum)
+
+        module ActiveRecordTypedEnum
```

Comment:
There are NOT specs for this Runtime code. I am trying to add them, but there isn't an existing pattern to follow for these Railties tests. And it needs a full Rails app to test, so having trouble fitting it into the existing `minitest` test suite

I plan on working on this, and should have specs for when we go to upstream this work. But was thinking that using this in the app and testing well in Stage should give us confidence in the Runtime code! We can also add some simple test cases in the app, since we will have our full Rails app available there!

Reply:
Having small smoke tests in the rails app side sounds good to me!